memcached.c

```diff
@@ -3911,7 +3918,7 @@ static inline void process_get_command(conn *c, token_t *tokens, size_t ntokens,
                 goto stop;
             }
 
-            it = limited_get(key, nkey, c, exptime, should_touch, DO_UPDATE, &overflow);
+            it = limited_get(key, nkey, c, realtime(exptime), should_touch, DO_UPDATE, &overflow);
```

Comment:
move the realtime() conversion back to the if (should_touch) section. no reason to keep converting it.